Sources/MapGL/MapView.swift

```diff
@@ -311,6 +345,49 @@ public class MapView : UIView {
 		self.js.fetchGeographicalBounds(completion: completion)
 	}
 
+	/// Upload styles object by its id and apply it to the map.
+	/// - Parameters:
+	///   - style: Uuid of the style.
+	public func setStyle(style: String) {
+		_style = style
+		self.js.setStyle(style: style)
+	}
+
+	/// Sets whole map style global variables at once, any previously set variables will be reset or overriden.
+	public func setStyleState(styleState: [String: Bool]) {
+		self.js.setStyleState(styleState: styleState)
+	}
+
+	/// Patches map style global variables. Use this method if you want to change a particular variable and leave other ones intact.
+	public func patchStyleState(styleState: [String: Bool]) {
+		self.js.patchStyleState(styleState: styleState)
+	}
+
+	/// Pans and zooms the map to contain its visible area within the specified geographical bounds.
+	/// This method also resets the map pitch and rotation to 0. But the map rotation can be saved
+	/// by option considerRotation.
+	/// - Parameters:
+	///   - bounds: The geographical bounds to fit in.
+	///   - options: FitBounds options.
+	public func fitBounds(bounds: GeographicalBounds, options: FitBoundsOptions? = nil) {
+		self.js.fitBounds(bounds: bounds, options: options)
+	}
+
+	/// Tests whether the current browser supports MapGL.
+	/// Use our raster map implementation https://api.2gis.ru/doc/maps/en/quickstart/ if not.
+	public func isSupported(options: MapSupportOptions? = nil) -> Bool {
+		return notSupportedReason(options: options) == nil
```

Comment:
У нас принято писать `self` всегда.

```suggestion
		return self.notSupportedReason(options: options) == nil
```